src/condition.js

```diff
@@ -84,7 +84,11 @@ export default class Condition {
    */
   _getValue (almanac) {
```

Comment:
just a note that I'd be peaceful with deleting this method as a non-breaking change since all callsites have been removed and the leading `_` denotes it as a private method